tests/transports-governance/test_helpers.py

```
@@ -0,0 +1,644 @@
+
+        if include_rate_limit:
+            data["rate_limit"] = TestDataFactory.generate_rate_limit_config()
+
+        return data
+
+
+class ValidationHelper:
+    """Helper functions for validating test results"""
+
+    @staticmethod
+    def validate_entity_structure(
```

Ruff (0.12.2)

`validate_entity_structure` is too complex (13 > 10)

(C901)